robotbase/src/main/java/frc/robot/commands/rumble/RumbleCodriverController copy.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot.commands.rumble;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.Robot;
+
+public class RumbleCodriverController extends SequentialCommandGroup {
+  public RumbleDriverCodontroller(double intensity, double seconds) {
```

Comment:
If you are going to separate the commands to coDriver and Driver, I would also hardcode the intensity and duration constants into the command.